packages/client/src/pages/admin/index.vue

```diff
@@ -62,7 +62,7 @@ let view = $ref(null);
 let el = $ref(null);
 let pageProps = $ref({});
 let noMaintainerInformation = isEmpty(instance.maintainerName) || isEmpty(instance.maintainerEmail);
-let noBotProtection = !instance.enableHcaptcha && !instance.enableRecaptcha;
+let noBotProtection = instance.disableRegistration ? false : !instance.enableHcaptcha && !instance.enableRecaptcha;
```

Comment:
I think using a ternary expression here is unnecessary. You can use the logical and operator with short circuiting to the same effect:

```suggestion
let noBotProtection = !instance.disableRegistration && !instance.enableHcaptcha && !instance.enableRecaptcha;
```